qemu/tests/qemu_guest_agent.py

```diff
@@ -1633,6 +1633,66 @@ def check_vss_info(cmd_type, key, expect_value):
                 test.error("guest-fsfreeze-thaw cmd failed with:"
                            "('%s')" % str(detail))
 
+    @error_context.context_aware
+    def gagent_check_fsinfo(self, test, params, env):
+        """
+        Execute "guest-fsinfo" command to guest agent,check file system of
+        mountpoint,disk's name and serial number.
+
+        steps:
+        1) check file system of every mount point.
+        2) check disk name.
+        3) check disk's serial number.
+
+        :param test: kvm test object
+        :param params: Dictionary with the test parameters
+        :param env: Dictionary with test environment.
+
+        """
+        session = self._get_session(params, None)
+        self._open_session_list.append(session)
+
+        error_context.context("Check all file system info in a loop.", logging.info)
+        fs_info_qga = self.gagent.get_fsinfo()
+        for fs in fs_info_qga:
```

Comment:
Better do not use `fs` here, as `fs`, `disk_name`,`serial` are three check point.

Reply:
modify fs to fs_type.